x-pack/filebeat/module/zoom/webhook/_meta/fields.yml

```
@@ -0,0 +1,647 @@
+      description: >
+        The configured agenda of the webinar
+    - name: webinar.password
+      type: keyword
+      description: >
+        Password configured to access the webinar
+    - name: webinar.issues
+      type: keyword
+      description: >
+        Any reported issues about a webinar is reported in this field
+    - name: zoomroom.id
+      type: keyword
+      description: >
+        Unique ID of the Zoom room
+    - name: zoomroom.room_name
+      type: keyword
+      description: >
+        The configured name of the Zoom room
+    - name: zoomroom.calendar_name
+      type: keyword
+      description: >
+        Calendar name of the Zoom room
+    - name: zoomroom.calendar_id
+      type: keyword
+      description: >
+        Unique ID of the calendar used by the Zoom room
+    - name: zoomroom.event_id
+      type: keyword
+      description: >
+        Unique ID of the calendar event associated with the Zoom Room
+    - name: zoomroom.change_key
+      type: keyword
+      description: >
+        Key used by Microsoft products integration that represents a specific version of a calendar
+    - name: zoomroom.resource_email
+      type: keyword
+      description: >
+        Email address associated with the calendar in use by the Zoom room
+    - name: zoomroom.email
+      type: keyword
+      description: >
+        Email address associated with the Zoom room itself
+    - name: zoomroom.issue
+      type: keyword
+      description: >
+        Any reported alerts or issues related to the Zoom room or its equipment
+    - name: zoomroom.alert_type
+      type: keyword
+      description: >
+        A integer value representing the type of alert. The list of alert types can be found in the Zoom documentation
+    - name: zoomroom.component
+      type: keyword
+      description: >
+        A integer value representing the type of equipment or component, The list of component types can be found in the Zoom documentation
+    - name: zoomroom.alert_kind
+      type: keyword
+      description: >
+        A integer value showing if the Zoom room alert has been either 1(Triggered) or 2(Cleared)
+    - name: registrant.id
+      type: keyword
+      description: >
+        Unique ID of the user registrating to a meeting or webinar
+    - name: registrant.status
+      type: keyword
+      description: >
+        Status of the specific user registration
+    - name: registrant.email
+      type: keyword
+      description: >
+        Email of the user registrating to a meeting or webinar
+    - name: registrant.first_name
+      type: keyword
+      description: >
+        First name of the user registrating to a meeting or webinar
+    - name: registrant.last_name
+      type: keyword
+      description: >
+        Last name of the user registrating to a meeting or webinar
+    - name: registrant.address
+      type: keyword
+      description: >
+        Address of the user registrating to a meeting or webinar
```

```suggestion
        Address of the user registering to a meeting or webinar
```

Make sure to adjust the other instances of the word above and below :-)